clients/src/main/java/org/apache/kafka/common/metrics/ClientTelemetryReceiver.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kafka.common.metrics;
+
+import org.apache.kafka.common.annotation.InterfaceStability;
+
+@InterfaceStability.Evolving
+public interface ClientTelemetryReceiver {
+    /**
+     * Called by the broker when a client reports telemetry. The associated request context can be used
+     * by the plugin to retrieve additional client information such as client ids or endpoints.
+     *
+     * This method may be called from the request handling thread, and as such should avoid blocking.
+     *
+     * @param context the client request context for the corresponding PushTelemetryRequest api call
+     * @param payload the encoded telemetry payload as sent by the client
+     */
+    void exportMetrics(org.apache.kafka.server.authorizer.AuthorizableRequestContext context, ClientTelemetryPayload payload);
```

Comment:
Can this be changed from `org.apache.kafka.server.authorizer.AuthorizableRequestContext` to `AuthorizableRequestContext` by adding a corresponding `import` statement?

Reply:
For some vague reason I am getting checkStyle error if I import the org.apache.kafka.server.authorizer.AuthorizableRequestContext, so this is the workaround to fix that error. Still I am not sure why our checkStyle script complains about this import.

Reply:
Ah, interesting. Is it an error something about not being allowed to import `AuthorizableRequestContext` into `ClientTelemetryReceiver`?

Reply:
Yeah some thing like that..

Reply:
[2022-03-23T07:08:19.798Z] [ant:checkstyle] [ERROR] /home/jenkins/workspace/confluentinc-pr_kafka_PR-683/clients/src/main/java/org/apache/kafka/common/metrics/ClientTelemetryReceiver.java:21:1: Disallowed import - org.apache.kafka.server.authorizer.AuthorizableRequestContext. [ImportControl]